### DISCUSSION TITLE
Problem with 20230630 Update ?

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1376] [Bug Reports]
I applied the 20230630 update to my RPi3A this morning and after copying the new files the update ended with:

/srv/http/bash/settings/player-conf.sh: line 17: 17894 Killed                  $dirbash/status-push.sh
/srv/http/bash/common.sh: line 152: echo: write error: Broken pipe
/srv/http/bash/status.sh: line 20: echo: write error: Broken pipe

Is this a problem ?

Thanks

Chris

**Comment by kw-sudo** [comment]
Update RPi3B+ (existing 20230620 installation with a complete pacman -Syu, config settings manually)

- Library view issue still exists
- 20230630 addon update ended with:
- `amixer: Unable to find simple control 'nubert xCore USB Audio',0
amixer: Unable to find simple control 'nubert xCore USB Audio',0
`

Works anyway.

PS: in i20230521 + pacman -Syu no issue

Oliver

**Reply by kw-sudo** [reply]
Because it is so nice, another pacman -Syu and complete library rebuild. :-D

[drum roll] Library view (album) is fine again, even in "sort by artist" combined with "sort by year". 

Thx @rern !

**Reply by kw-sudo** [reply]
Adding full album songs are shown, not hidden anymore

**Comment by ceejayemm** [comment]
Applying the current UPDATE branch (as at 09:47 BST 03/07/23) still gives the broken pipe errors I originally noted above.

Chris

**Reply by rern** [reply]
Hi Chris,

Post the output of each script:
- player-conf.sh
- common.sh
- status.sh

**Comment by ceejayemm** [comment]
@rern 
As requested:

rAudio-1:~  #  player-conf.sh
/srv/http/bash/common.sh: line 194: /srv/http/data/shm/pidstatuspush: No such file or directory
rAudio-1:~  #  

common.sh - no output

rAudio-1:~  #  status.sh
{ 
  "player"       : "mpd"
, "btreceiver"   : false
, "card"         : 0
, "consume"      : false
, "control"      : "PCM"
, "counts"       : {
  "album": 337,
  "albumartist": 132,
  "albumyear": 608,
  "artist": 649,
  "composer": 1052,
  "conductor": 2,
  "dabradio": 0,
  "date": 162,
  "genre": 38,
  "latest": 2,
  "nas": 1,
  "playlists": 0,
  "sd": 0,
  "song": 5106,
  "usb": 0,
  "webradio": 26
}
, "icon"         : ""
, "librandom"    : false
, "lyrics"       : false
, "relays"       : false
, "relayson"     : false
, "scrobble"     : false
, "shareddata"   : true
, "snapclient"   : false
, "stoptimer"    : false
, "updateaddons" : false
, "updating_db"  : false
, "updatingdab"  : false
, "volume"       : 86
, "volumemute"   : 0
, "repeat" : false
, "random" : false
, "single" : false
, "file"      : "http://a.files.bbci.co.uk/media/live/manifesto/audio/simulcast/hls/uk/sbr_high/ak/bbc_radio_two.m3u8"
, "pllength"  : 1
, "song"      : 0
, "timestamp" : 1688375659154
, "stream" : true
, "Album"        : ""
, "Artist"       : ""
, "stationcover" : "/data/webradio/img/http:||a.files.bbci.co.uk|media|live|manifesto|audio|simulcast|hls|uk|sbr_high|ak|bbc_radio_two.m3u8.jpg"
, "Name"         : ""
, "state"        : "stop"
, "station"      : "BBC Radio 2"
, "Time"         : false
, "Title"        : ""
, "webradio"     : true
, "ext"      : "Radio"
, "coverart" : ""
, "icon"     : "webradio"
, "sampling" : "1/1 •  kHz 0 kbit/s"
, "elapsed"  : 7084
, "coverart" : ""
, "login" : false }
rAudio-1:~  #  


I hope this helps

Chris

**Comment by ceejayemm** [comment]
@rern 

I think player-conf.sh is telling lies !, see:

rAudio-1:~  #  ls -lrt /srv/http/data/shm
lrwxrwxrwx 1 http http 8 Jun 21 10:11 /srv/http/data/shm -> /dev/shm
rAudio-1:~  #  ls -lrt /srv/http/data/shm/
total 24
drwxrwxrwx 2 root root  40 Jun 26 16:55 webradio
drwxrwxrwx 2 root root  40 Jun 26 16:55 spotify
drwxrwxrwx 2 root root  40 Jun 26 16:55 online
drwxrwxrwx 2 root root  40 Jun 26 16:55 local
drwxrwxrwx 2 root root  40 Jun 26 16:55 embedded
drwxrwxrwx 2 root root  40 Jun 26 16:55 airplay
-rwxrwxrwx 1 root root  50 Jun 26 16:58 cpuinfo
-rwxrwxrwx 1 root root   6 Jun 26 16:58 wlan
-rwxrwxrwx 1 root root   0 Jun 26 16:58 onboardwlan
-rwxrwxrwx 1 root root   0 Jun 26 16:58 startup
drwxrwxrwx 2 root root  60 Jun 26 17:02 sampling
-rwxrwxrwx 1 root root   4 Jul  1 12:14 player
-rw-r--r-- 1 root root   0 Jul  3 10:13 mixernone
-rw-r--r-- 1 root root   6 Jul  3 10:13 pidstatuspush
-rw-r--r-- 1 root root 246 Jul  3 10:13 status
-rwxrwxrwx 1 root root   4 Jul  3 10:13 amixercontrol
rAudio-1:~  #  

Running it a second time gives:

rAudio-1:~  #  ls -lrt /srv/http/data/shm/pidstatuspush
-rw-r--r-- 1 root root 6 Jul  3 10:13 /srv/http/data/shm/pidstatuspush
rAudio-1:~  #  player-conf.sh
/srv/http/bash/settings/player-conf.sh: line 17: 29874 Killed                  $dirbash/status-push.sh
rAudio-1:~  #

**Comment by ceejayemm** [comment]
Any progress with this issue please ?

Thanks

Chris
